templates/taxi/driver_form.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <h1>{{ object|yesno:"Update,Create" }} driver</h1>
+  <form action="" method="post" novalidate>
+    {% csrf_token %}
+    {{ form|crispy }}
+
+    <input type="submit" value="Submit" class="btn btn-primary">
+    {% if object %}
+      <a class="btn btn-danger" href="{% url 'taxi:driver-detail' pk=driver.id %}">Cancel</a>
```

Comment:
The variable `driver` is not defined in this template. It should be `object` if you are using the generic views context. Change `pk=driver.id` to `pk=object.id` to ensure it works correctly.